apps/fz_http/priv/repo/migrations/20221224125052_users_fk_delete_all_constraint.exs

```diff
@@ -0,0 +1,17 @@
+defmodule FzHttp.Repo.Migrations.UsersFkDeleteAllConstraint do
+  use Ecto.Migration
+
+  def change do
+    drop(constraint(:oidc_connections, "oidc_connections_user_id_fkey"))
+
+    alter table(:oidc_connections) do
+      modify(:user_id, references(:users, on_delete: :delete_all), null: false)
```

Comment:
Just something to be aware of: this will be a conflict on rebase, you will need to rename the migration timestamp to be after this one (to guarantee clean order on fresh dev environments) and update type `type: :binary_id` in UUID migrations I pushed.